src/App.tsx

```diff
@@ -1,39 +1,99 @@
-import React from 'react';
+import React, { useState } from 'react';
+
 import './App.scss';
-import Carousel from './components/Carousel';
-
-interface State {
-  images: string[];
-}
-
-class App extends React.Component<{}, State> {
-  state = {
-    images: [
-      './img/1.png',
-      './img/2.png',
-      './img/3.png',
-      './img/4.png',
-      './img/5.png',
-      './img/6.png',
-      './img/7.png',
-      './img/8.png',
-      './img/9.png',
-      './img/10.png',
-    ],
+import './services/reset.scss';
+import { Carousel } from './components/Carousel';
+import { CarouselParameters } from './types/CarouselParameters';
+import { getNumbers } from './services/utils';
+
+const defaultCarouselParameters: CarouselParameters = {
+  step: 3,
+  frameSize: 3,
+  itemWidth: 130,
+  animationDuration: 1000,
+};
+
+const getImagesUrls = getNumbers(1, 10)
+  .map(n => `./img/${n}.png`);
+
+export const App: React.FC = () => {
+  const [carouselParameters, setCarouselParameters] = (
+    useState<CarouselParameters>({ ...defaultCarouselParameters })
+  );
+
+  const images = getImagesUrls;
+  const isInfinite = false;
+
+  const handleFieldChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setCarouselParameters({
+      ...carouselParameters,
+      [event.target.name]: +event.target.value,
+    });
```

Comment:
```suggestion
   const { name, value} = event.target;
   
    setCarouselParameters({
      ...carouselParameters,
      [name]: +value,
    });
```
you can ditrructure event.target